test/LocatedTriplesTest.cpp

```diff
@@ -833,7 +833,7 @@ TEST_F(LocatedTriplesTest, augmentedMetadataGraphInfo) {
       1, 10, 10,
       12);  // Before block 0 (because `12` is smaller than the default graph)
   auto T2 = IT(1, 10, 10,
-               99999999);  // Becomes the lower bound of block 1, although it
+               99999999);  // Becomes the last triple of block 0, because it
                            // only differs in the graph info.
```

Comment:
If we put the comments in separate lines, the formatting becomes nicer since this should avoid the weird line breaks.